assets/js/modules/analytics-4/index.js

```diff
@@ -989,100 +867,39 @@ export const ANALYTICS_4_NOTIFICATIONS = {
 			VIEW_CONTEXT_MAIN_DASHBOARD_VIEW_ONLY,
 		],
 		isDismissible: true,
-		checkRequirements: async ( { select, resolveSelect }, viewContext ) => {
-			const ga4ModuleConnected = await resolveSelect(
-				CORE_MODULES
-			).isModuleConnected( MODULE_SLUG_ANALYTICS_4 );
-			const ga4ModuleActive = await resolveSelect(
-				CORE_MODULES
-			).isModuleActive( MODULE_SLUG_ANALYTICS_4 );
-
-			// If the module is not connected or not active, we can return early
-			// to prevent additional audience settings being requested.
-			if ( ! ga4ModuleConnected || ! ga4ModuleActive ) {
-				return false;
-			}
-
-			const isViewOnly =
-				SITE_KIT_VIEW_ONLY_CONTEXTS.includes( viewContext );
```

Comment:
I chose to replace this with is-authenticated rather than basing on viewContext.